README.md

```diff
@@ -121,6 +122,27 @@ To stop a node:
 $ mn stop
 ```
 
+### Restart node
+
+The `restart` command is used to restart a node with the default or specified config.
+
+```
+USAGE
+  $ mn restart
+
+OPTIONS
+  -f, --full-node                                  start as full node
```

Comment:
This kinda strange to see such a flag in restart command. Perhaps, we should move it to config?

Reply:
ok, moved to config